PLEP-0007.rst

```diff
@@ -0,0 +1,266 @@
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of functions, classes, and sub-packages dedicated to solving  |
+|   various plasma dispersion relations.                                       |
+|                                                                              |
+|   All dispersion solvers defined in this sub-package should be accessible    |
+|   through a dispersion solver factory class, ``DispersionSolver``.           |
++------------------------------------------------------------------------------+
+| .. _formulary:                                                               |
+|                                                                              |
+| **formulary**                                                                |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/formulary/``                                        |
+| :access: ``plasmapy.formulary``                                              |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of functions, classes, and sub-packages that provide          |
+|   mathematical and scientific formulas for calculating physical parameters   |
+|   of various plasmas.                                                        |
++------------------------------------------------------------------------------+
+| .. _particles:                                                               |
+|                                                                              |
+| **particles**                                                                |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/particles/``                                        |
+| :access: ``plasmapy.particles``                                              |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of functions, classes, and sub-packages that fully defines    |
+|   the properties of a "particle."  A "particle" can come in many forms       |
+|   ranging from a traditional particle (electron, ion, atom, etc.) to more    |
+|   exotic types like dust particles, dimensionless particles for simulations, |
+|   super-particles for simulations, etc.                                      |
+|                                                                              |
+|   All types of particles in this sub-package should be able to be            |
+|   instantiated through a particle factory class, ``Particle``.               |
```

Comment:
I'm not certain that we want to specify this much detail here.  I'm wondering if we will end up having a separate `CustomParticle` class to represent user-defined particles, as that would require a lot of different arguments than for a typical `Particle` instance.  My inclination is for this PLEP to be agnostic about the implementation.